test/test_validation.py

```diff
@@ -463,6 +463,30 @@ def test_invalid_items(self):
             'Incorrectly accepts integers outside of the range'
         )
 
+   def test_valid_character_items(self):
```

Comment:
The Travis error is caused by the fact that you've only indented this with 3 spaces. It should be 4 spaces per indentation